Nos/Service/FeatureFlags.swift

```diff
@@ -1,17 +1,25 @@
 import Foundation
 import Dependencies
 
-/// The set of feature flags used by the app.
-protocol FeatureFlags {
+/// Enum to represent each feature flag. Add new feature flags here easily.
+enum FeatureFlag: String {
```

Comment:
There's currently no reason for this enum to be typed to `String`.
```suggestion
enum FeatureFlag {
```